### ISSUE TITLE
Immediate detumescence of penis at time of symptom onset

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10013]
**Preferred term label:**
Immediate detumescence of penis at time of symptom onset

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Penile fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
it seems this phenotype is quite specific for penile fracture, so there may not be much gain by adding a term. Instead, we could add this to the description of a term Penile fracture?

**Comment by MickeySegal** [comment]
That approach doesn't fit well with clinical practice.  A patient doesn't give a history of "I have a penile fracture".  The patient gives the history of "I was having rough sex, felt pain, lost my erection immediately, and now my penis is swollen and at an angle".  Those should all be coded separately, and several already have HPO codes.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000982
- label: Immediate detumescence of penis at time of symptom onset
- parents: [Abnormality of the male genitalia (HP:0010461)](https://hpo.jax.org/app/browse/term/HP:0010461)
- pmid: 
- def: Loss of erection at the time point of pain or other symptom onset.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10013
- synonyms:
